proto/rill/local/v1/api.proto

```diff
@@ -28,6 +28,8 @@ service LocalService {
 
   // User returns the locally logged in user
   rpc GetCurrentUser(GetCurrentUserRequest) returns (GetCurrentUserResponse) {}
+
+  rpc CheckOrgName(CheckOrgNameRequest) returns (CheckOrgNameResponse) {}
```

Comment:
I don't know the full e2e flow from UI but for CLI we are mostly creating first and if conflict exist then prompting for project/org names. The org name can still be taken if this check passes plus it avoids extra network call.

Reply:
What could be reason for org name to exist even if this passes? Race condition?

We do not have a flow to prompt user in UI. It has to automatically select the next available by adding `-1`, `-2` suffixes.

Reply:
> What could be reason for org name to exist even if this passes? Race condition?

Yes

> It has to automatically select the next available by adding -1, -2 suffixes.

Then I would suggest lets create first and if it fails due to name mismatches then select the next available ?